src/modules/drawing/DrawingModule.vue

```diff
@@ -147,6 +147,12 @@ export default {
             }
             return null
         },
+        isNewDrawing() {
```

Comment:
Good catch I changed it

Reply:
However this is not the issue of my test failing. The test is failing because the `this.kmlIds` is missing the `adminId` and create a new drawing instead of updating the existing. When doing the test manually it works. The issue is probably a timing issue with the kmlIds being on a store and updated in several places in the code. I simplified this logic in #326